helm/charts/hpe-csi-driver/values.schema.json

```diff
@@ -21,7 +21,7 @@
             "registry": "quay.io",
             "kubeletRootDir": "/var/lib/kubelet/",
             "disableNodeGetVolumeStats": false,
-            "cspClientTimeout": 60
+            "csiAttacherTimeout": 180
```

Comment:
Should this be defaulted to 15 s as well

Reply:
modified